conf/configuration.go

```diff
@@ -356,9 +357,30 @@ func updateConfigFromClowder(c *ConfigStruct) error {
 		fmt.Println(noStorage)
 	}
 
+	// get in-memory DB configuration from clowder
+	if !c.Redis.UseClowder {
+		fmt.Println("Clowder for Redis is disabled")
+		return nil
+	}
+	if clowder.LoadedConfig.InMemoryDb != nil {
+		updateRedisConfig(&c.Redis)
+	} else {
+		fmt.Println(noInMemoryDB)
+	}
+
 	return nil
 }
 
+func updateRedisConfig(conf *storage.RedisConfiguration) {
+	conf.RedisEndpoint = fmt.Sprintf("%s:%d", clowder.LoadedConfig.InMemoryDb.Hostname, clowder.LoadedConfig.InMemoryDb.Port)
+	if clowder.LoadedConfig.InMemoryDb.Username != nil {
```

Comment:
shall we return an error? if the clowder config has no username/password there's definitely something wrong

Reply:
Actually, ephemeral instance does not set username or password. There's no auth.